core/src/main/scala/org/apache/spark/sql/rapids/tool/ClassWarehouse.scala

```diff
@@ -26,7 +24,7 @@ class ExecutorInfoClass(val executorId: String, _addTime: Long) {
     var isActive = true
     var totalCores = 0
 
-    val addTime = new Date(_addTime)
+    val addTime = _addTime
```

Comment:
nit: Since we are directly copying the value, we could probably replace `_addTime` by `val addTime`